seed/challenges/01-front-end-development-certification/basic-ziplines.json

```diff
@@ -218,9 +218,9 @@
       ],
       "titleEs": "Construye una página web para tu portafolio",
       "descriptionEs": [
-        "<strong>Objetivo:</strong> Crea una aplicación con <a href='http://codepen.io' target='_blank'>CodePen.io</a> cuya funcionalidad sea similar a la de esta: <a href='http://codepen.io/FreeCodeCamp/full/VemmoX/' target='_blank'>http://codepen.io/FreeCodeCamp/full/VemmoX/</a>.",
+        "<strong>Objetivo:</strong> Crea una aplicación con <a href='https://codepen.io' target='_blank'>CodePen.io</a> cuya funcionalidad sea similar a la de esta: <a href='https://codepen.io/FreeCodeCamp/full/VemmoX/' target='_blank'>https://codepen.io/FreeCodeCamp/full/VemmoX/</a>.",
```

Comment:
Change to https://codepen.io/hallaathrad/full/vNEPpL as well? Uses the same `/VemmoX/` link